skfem/mapping/mapping_isoparametric.py

```diff
@@ -8,6 +8,10 @@
 from .mapping import Mapping
 
 
+NEWTON_MAX_ITERS = 50
```

Comment:
Good idea replacing the 'magic numbers' 50 and 1e-6 with variables.  

Should they be defined closer (structurally) to where they're used?  And they're only used in one method, so what about having them as optional keyword arguments (which would never in practice be used); e.g.

```python
def invF(self, x, tind=None, newton_max_iters: int=50, newton_tol=1e-8)
```

Reply:
Made the constants method local.